src/styles/index.scss

```diff
@@ -1,3 +1,89 @@
+@import './utils/variables';
+
 body {
   margin: 0;
+  height: 100vh;
```

Comment:
According to the code style rules, it's recommended to use relative units like 'em' or 'rem' for setting dimensions and spacing to ensure better scalability and maintainability of the styles. Consider replacing 'vh' units with 'em' or 'rem' for the 'height' property.